### PR TITLE
docs(reports): 2024 Annual Hyperledger Bevel Report

### PR DESCRIPTION
**Pull request by suvajit-sarkar**
Combined report for Bevel and Bevel-Operator-Fabric

### CODE REVIEW

File: project-reports/2024/2024-annual-Hyperledger-Bevel.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+[Hyperledger Bevel LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=bevel&repository=&dateFilters=Last%20Year&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true)
+
+[Hyperledger Bevel Operator Fabric LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=bevel&repository=https:%2F%2Fgithub.com%2Fhyperledger%2Fbevel-operator-fabric)
+
+1. Getting code contribution on Bevel still remains a challenge, though the community usage and interest remains positive for both the projects as can be inferred from discussions and queries on discord channel and user participations on meetups.
+3. The projects have done well in terms of its release commitments.
+4. Bevel recently came out with its general availability release v1.0, with focus on seamless deployment. 
+5. Bevel-Operator-Fabric recently came out with its release v1.10.0, with a focus on stability of network deployment.
+6. Bevel has accepted a major code contribution. (helm charts for deploying Hyperledger Aries).
```
Comment (swcurran) [inline project-reports/2024/2024-annual-Hyperledger-Bevel.md:18]: Interesting stuff.  I looked a bit to see where this was and didn’t find, but I’m interested in this.  Where is the contribution coming from?  What does it cover?  Thanks!

File: project-reports/2024/2024-annual-Hyperledger-Bevel.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+### Goal 2024: Growing the Community
+- Workshop and demos on upcoming changes and features
+
+## Help Required
+
+- Continue to work with Hyperledger staff to identify contributors and potential maintainers outside current maintainers and orgs.
+- Mention Hyperledger Bevel as a means for deployment (on Kubernetes) on documentations’ of Hyperledger projects such as Fabric, Besu, Indy and Aries.
+
+# Project Lifecycle Stage Recommendation
+Currently in `incubation`, has potential to be `Graduated`.
```
Comment (swcurran) [inline project-reports/2024/2024-annual-Hyperledger-Bevel.md:87]: Is this a request/recommendation to move to graduated or more of an observation?  It sounds like the latter.

File: project-reports/2024/2024-annual-Hyperledger-Bevel.md (hunk 1 above)
Comment (suvajit-sarkar) [reply project-reports/2024/2024-annual-Hyperledger-Bevel.md:18]: Hi Stephen, the contribution comes from user monar24 from superlogic team. It covers helm charts for deploying agents using ACA-Py on Kubernetes. You can check the PR and follow the conversations around it [here](https://github.com/hyperledger/bevel/pull/2501).

File: project-reports/2024/2024-annual-Hyperledger-Bevel.md (hunk 2 above)
Comment (suvajit-sarkar) [reply project-reports/2024/2024-annual-Hyperledger-Bevel.md:87]: Yeah now that you have asked, it does sound like an observation. But what we meant to convey is that, we need to do some more work (one more major release) before we can confidently request/recommend for the project graduation.

**Comment by swcurran** [comment]
## **Hyperledger Bevel 2024 Annual Review Notes (from Stephen and Marcus)**


### **Project Health**



* [Insights data](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=bevel&repository=&dateFilters=Last%20Year&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true)
    * Generally positive metrics for the year 2023, with moderate increases in each of commits, issues, pull requests, and forks. Down a bit (~7%) on contributors (overall and new), as mentioned in the Annual Report.
        * I think more telling is the goals set for 2023 and their achievement, later in the report.
    * Contributor dependency was noted in the report as a challenge, and that is born out in the metrics
        * 55% of the code activities performed by 4 people, and about 42% by two people.
        * 75% of the activity from the lead organization on the project, Accenture. That said, there were contributions from a total of 35 organizations over the year.


### **Maintainer Diversity**



* There are maintainers from three organizations, with the three organizations topping the organizational activity leaderboard. Nice to have a few more maintainers from other entities.


### **Project Adoption**



* Adoption information looks good. Project is in use by a number of different enterprises.


### **Goals**



* 2023 goals were completed — nice work on completing the v1.0 release, and the non-code efforts on growing the community.
* 2024 goals have been defined that seem consistent with what I have been hearing from the Indy/Aries community — the shift to using Helm charts, and Kubernetes deployments.


### **Help Required**



* While there is a general request for the other projects to update their documentation to include Bevel, it might be a good idea for the Bevel maintainers to proactively add pointers to Bevel capabilities in the documentation of the other project via pull requests.  Even a couple of lines in the readme, including a pointer into the relevant (nice!) Bevel documentation might be useful. That might also trigger invites for Bevel experts to present at a Project call/meetup.
* Does the project see contributions from other project maintainers to enable new platforms/features? For example, do you see help from Fabric maintainers to integrate Fabric 3.0?
* There is a need to increase the maintainer diversity on the project. The project is looking for help on this. Since most of our projects need help on this, would it be worth putting together a task force to document guidelines for how to increase maintainer diversity?


### **Project Lifecycle Stage Recommendation**



* Project recommends remaining in the incubation stage. Continuing to increase the maintainer diversity would help in enabling the transition to “graduated.”


### **Other Notes**



* Conformance to the [repository structure guidelines](https://github.com/hyperledger/toc/blob/gh-pages/guidelines/repository-structure.md) - (for core repository)
    * `README.md` and docs look really good, nice job!
    * Missing `SECURITY.md` file and the CHANGELOG.md has a different name.
    * `MAINTAINERS.md` doesn't include scope per the suggested [format](https://github.com/hyperledger/toc/blob/gh-pages/guidelines/MAINTAINERS-guidelines.md) (most projects haven't done this yet)
    * No `NOTICE` file (Note to TOC - is this really important?)
    * CONTRIBUTING.md contains some broken links
    * Other than the notes in the Annual Report, is there a Bevel Roadmap?

**Review by swcurran — APPROVED** [review]

**Review by tkuhrt — APPROVED** [review]

**Pull request merged**